### PR TITLE
Mantine demos

### PR DESCRIPTION
**Pull request by GermainBergeron**
### Proposed Changes

Added a script that fetch the demo from Mantine's GitHub and generate pages for us to showcase our theme

You should be able to see the pages in Mantine section of the menu. I had to skip over some demo because they are broken (missing import, missing prop, wrong prop, etc)

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/bin/index.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import simpleGit, {SimpleGit} from 'simple-git';
+import {DEMO_FILENAME_PATTERN} from './constants';
+import {copyDemoFiles} from './copyDemoFiles';
+import {findTsxAndCssModule} from './findTsxAndCssModule';
+import {generatePage} from './generatePage';
+import {groupFilesByComponent} from './groupFilesByComponent';
+
+const cloneRepository = async (repoUrl: string, destinationFolder: string): Promise<void> => {
+    const git: SimpleGit = simpleGit();
+    await git.clone(repoUrl, destinationFolder);
```
Comment (gdostie) [inline packages/website/bin/index.ts:12]: Should we add clone to [semantic-monorepo-tools](https://github.com/coveo/semantic-monorepo-tools/tree/main/src/git)?

**Review by gdostie — APPROVED** [review]
A good start 👍 There are stuff we could cleanup in each page like remove the empty props table or add a link to the actual mantine website page, but we can do it later on.

**Review by dmgauthier — APPROVED** [review]
Great stuff it works super well

File: packages/website/bin/index.ts (hunk 1 above)
Comment (GermainBergeron) [reply packages/website/bin/index.ts:12]: I'll create tasks so we don't forget but I wouldn't block this PR for that if you agree

**Pull request merged**